examples/classic_controllers/classic_controllers.py

```diff
@@ -900,8 +1632,8 @@ def reset(self):
     'pid_controller': [ContinuousActionController, ContinuousController, PID_Controller],
```

Comment:
Also these class names violate the PEP8 conventions. The recommended naming would be PIDController. Please change it also for all remaining classes (PController / PIController / ThreePointController etc.)